test/unit/aws/test_deploy_vm_bucket_waf.py

```diff
@@ -8,6 +8,7 @@
 from test.mock_cast import mock_cast
 
 
+# TODO: Do we need to generalize these tests apply them to Example Data S3 Bucket, too?
```

Comment:
Clarify or remove?

Reply:
I would like to clarify, please.

Reply:
I aligned with @tomuben and we decided to generalize the test cases.
See latest push.